tests/small/test_download_center.py

```diff
@@ -493,6 +493,20 @@ class WrongChecksumType(Enum):
         self.assertIsNone(result.fd)
         self.expect_warn_error = True
 
+    def test_download_timeout(self):
+        """we fail a download with timeout"""
+        filename = "simplefile"
+        url = self.build_server_address(filename)
+        request = DownloadItem(url, None, timeout=0.0000001)
```

Comment:
This seems a little bit racy, I would rather in the server add a real timeout issue.

Trigger that special behavior on <filename>-timeout for instance!